stories/round-tracker.stories.js

```diff
@@ -0,0 +1,112 @@
+// https://github.com/EnCiv/civil-pursuit/issues/101
+import React from 'react';
+import RoundTracker from '../app/components/round-tracker';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+
+export default {
+  title: 'RoundTracker',
+  component: RoundTracker,
+  parameters: {
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+  },
+};
+
+const Template = (args) => <RoundTracker {...args} />;
+
+export const OneRound = Template.bind({});
+OneRound.args = {
+  roundsStatus: ['inProgress', 'pending', 'pending'],
+};
+
+export const TwoRounds = Template.bind({});
+TwoRounds.args = {
+  roundsStatus: ['complete', 'inProgress', 'pending'],
+};
+
+export const ThreeRounds = Template.bind({});
+ThreeRounds.args = {
+  roundsStatus: ['complete', 'complete', 'inProgress', 'pending'],
+};
+
+export const FourRounds = Template.bind({});
+FourRounds.args = {
+  roundsStatus: ['complete', 'complete', 'complete', 'inProgress', 'pending'],
+};
+
+export const EightRounds = Template.bind({});
+EightRounds.args = {
+  roundsStatus: ['complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'inProgress', 'pending'],
+};
+
+export const TwelveRounds = Template.bind({});
+TwelveRounds.args = {
+  roundsStatus: ['complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'complete', 'inProgress'],
+};
+
+export const EmptyRounds = Template.bind({});
+EmptyRounds.args = {
+  roundsStatus: [],
+};
+
+// Mobile views
+export const MobileOneRound = Template.bind({});
+MobileOneRound.args = {
+  roundsStatus: ['inProgress', 'pending'],
+};
+MobileOneRound.parameters = {
+  viewport: {
+    defaultViewport: 'iphonex',
+  },
+};
+
```

Comment:
This method of setting args, and then setting parameters seems to be causing storybook to render twice, first with the args change, and second with the parameters change.  I didn't know this either.  But here is a shorter way to write the tests:

```js
// Mobile views
export const MobileOneRound = {
  args: {
    roundsStatus: ['inProgress', 'pending'],
  },
  parameters: {
    viewport: {
      defaultViewport: 'iphonex',
    },
  },
}
```
Please use this pattern for all the stories.    The Template= stuff is actually part of an older revision of Storybook, but they started doing it this way to make the code smaller, easier.